setup.py

```diff
@@ -59,6 +59,8 @@
         'invenio-sipstore>=1.0.0a1',
     ],
     'deposit': [
+        # Temporary pin to avoid collision with Flask-Security
```

Comment:
I think it's coming from invenio-oauth2server, that is required by invenio-deposit.